extensions/amp-embedly-card/amp-embedly-card.md

```diff
@@ -0,0 +1,114 @@
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="`amp-embedly-card`"></a> `amp-embedly-card`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Provides you with responsive and shareable embeds to drive the reach of your websites, blog posts, and articles. from any URL using <a href="http://docs.embed.ly/docs/cards">Embedly cards</a>.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-embedly-card-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
+    <td>responsive</td>
+  </tr>
+</table>
+
+## Behavior
+
+The `amp-embedly-card` component provides you with responsive and shareable embeds to drive the reach of your websites, 
+blog posts, and articles from any URL using <a href="http://docs.embed.ly/docs/cards">Embedly cards</a>.
+
+Cards are the easiest way to leverage Embedly. For any media, cards provide a responsive embed with built-in embed analytics.
+
+*Example: Embedding multiple resources*
+
+If you have a paid plan, use the `amp-embedly-key` component to set your api key. 
+You just need one per AMP page.
+
+```html
+<amp-embedly-key
+    value="12af2e3543ee432ca35ac30a4b4f656a"
+    layout="nodisplay">
+</amp-embedly-key>
+```
+
+Setting a key through the `amp-embedly-key` component is optional. If you are a paid user, this removes Embedly's branding from the cards.
+
+Within your AMP page, you can include one or multiple `amp-embedly-card` components:
+
+```html
+<amp-embedly-card
+    data-url="https://twitter.com/AMPhtml/status/986750295077040128"
+    layout="responsive"
+    width="150"
+    height="80"
+    data-card-theme="dark"
+    data-card-controls="0">
+</amp-embedly-card>
+
+<amp-embedly-card
+    data-url="https://www.youtube.com/watch?v=lBTCB7yLs8Y"
+    layout="responsive"
+    width="100"
+    height="50">
+</amp-embedly-card>
+```
+
+## Attributes
+
+##### data-url (required)
+  
+The URL to retrieve embedding information. 
+
+##### data-card-via
+
+Specifies the via content in the card. This is a a great way to do attribution.
+
+##### data-card-chrome
+
+Removes the left-hand colored border if the value is set to `0`.
+
+##### data-card-theme
+
+For dark backgrounds it's better to specify the dark theme.
+
+##### data-card-image
+
+Specifies which image to use in article cards.
+
+##### data-card-controls
+	
+Enables share icons. The default value is `1`.
```

Comment:
Clarify that by default share icons are enabled.  To turn them off, specify '0'.?